contracts/libraries/PlasmaVaultLib.sol

```diff
@@ -282,4 +283,10 @@ library PlasmaVaultLib {
     function isExecutionStarted() internal view returns (bool) {
         return PlasmaVaultStorageLib.getExecutionState().value == 1;
     }
+
+    function updateWithdrawManager(address newWithdrawManager_) internal {
```

Comment:
Add comment  / description that this address can be zero which means that schedule withdraw is turned off